File: Python/1116 - Dividing X by Y.py
n = int(input())
for i in range(n):
    s = input().split()
    for i in range(len(s)):
        s[i] = int(s[i])
    if (s[1] == 0):
        print("divisao impossivel")
    else:
        res = float((s[0]/s[1]))
        print("%.1f" % res)

File: C++/1116 - Dividing X by Y.cpp
#include <iostream>
#include <cstdio>
 
int main() {
    int n,x,y;
    std::cin >> n;
    for (int i = 0; i < n; i++)
    {
        std::cin >> x >> y;
        if (y == 0)
        {
            std::cout << "divisao impossivel\n";
        }
        else
        {
            printf("%.1f\n", (x/double(y)) );
        }
    }
    return 0;
}